adminer/lang/de.inc.php

```diff
@@ -280,7 +280,7 @@
 	'If you did not send this request from Adminer then close this page.' => 'Wenn Sie diese Anfrage nicht von Adminer gesendet haben, schließen Sie diese Seite.',
 	'You can upload a big SQL file via FTP and import it from server.' => 'Sie können eine große SQL-Datei per FTP hochladen und dann vom Server importieren.',
 	'You are offline.' => 'Sie sind offline.',
-	'You have no privileges to update this table.' => 'Sie haben keine Rechte, um diese Tabelle zu aktualisieren.',
+	'You have no privileges to update this table.' => 'Sie haben keine Rechte, diese Tabelle zu aktualisieren.',
```

Comment:
I think the existing translation is correct too. (a native German) 

Maybe 'Sie haben keine Berechtigung, um diese Tabelle zu aktualisieren.' is also possible.

Reply:
It sounds weird to any native German. That's why without 'um' it sounds more natural.

Reply:
https://www.duden.de/sprachwissen/sprachratgeber/um-zu-zu

Not weird for me. Maybe for you. Maybe regional, dialectical, age variation.

Reply:
I answered like I did because I spoke to some (2) native Germans in linguistics. We're all in agreement, that without 'um' sounds better.